### ISSUE TITLE
In a file, "ODM2PythonAPI/ODM2/models.py"

### ISSUE THREAD

**Issue by cdesyoun** [ODM2/ODM2PythonAPI#1]
Would you please fix these problems below?
  -- please fix the spelling error (ResultDateTimeUTCOfffset ==> ResultDateTimeUTCOffset)
  -- please add the "cv_relationshiptype" class
class CVRelationshipType(Base):
    __tablename__ = 'cv_relationshiptype'
    __table_args__ = {u'schema': 'odm2'}

    Term = Column('term', String(255), nullable=False)
    Name = Column('name', String(255), primary_key=True)
    Definition = Column('definition', String(1000))
    Category = Column('category', String(255))
    SourceVocabularyUri = Column('sourcevocabularyuri', String(255))
    def __repr__(self):
        return "<CV('%s', '%s', '%s', '%s')>" %(self.Term, self.name, self.Definition, self.Category)

  -- plead add the type, "Integer" in the class "SamplingFeatureExternalIdentifiers".
      "BridgeID = Column('bridgeid', Integer,  primary_key=True, nullable=False)"
  -- please replace "BIT" type with "Boolean" type in the class "Specimens".
      "IsFieldSpecimen = Column('isfieldspecimen', Boolean, nullable=False)"

**Comment by cdesyoun** [comment]
Great! Two problems below were found.
  -- in the class "Results", add the column name, "resulttypecv".
  -- in the class "RelatedActions", remove the column, "ActionTypeCV" which is not in the table.

**Comment by cdesyoun** [comment]
another one was found.
  -- in the class "RelatedFeatures", switch "SamplingFeatureObj" with "RelatedFeatureObj".

**Comment by sreeder** [comment]
@cdesyoun  I've made all of these changes except changing the Boolean to a Bit. Could you explain why you would like this change. BIT is db specific, and we are trying to keep the api generic across the b types. are you having errors when running with Boolean?

**Comment by cdesyoun** [comment]
@sreeder No, I meant changing the type in "Specimens" class below from "BIT" type to "Boolean" type in ODM2PythonAPI. 

class Specimens(Base):
    __tablename__ = u'specimens'
    __table_args__ = {u'schema': u'odm2'}

    SamplingFeatureID = Column('samplingfeatureid', ForeignKey('odm2.samplingfeatures.samplingfeatureid'), primary_key=True)
    SpecimenTypeCV = Column('specimentypecv', ForeignKey('odm2.cv_specimentype.name'), nullable=False, index=True)
    SpecimenMediumCV = Column('specimenmediumcv', ForeignKey('odm2.cv_specimenmedium.name'), nullable=False, index=True)
    #IsFieldSpecimen = Column('isfieldspecimen', BIT, nullable=False)
    IsFieldSpecimen = Column('isfieldspecimen', Boolean, nullable=False)
    SamplingFeatureObj = relationship(SamplingFeatures)

**Comment by cdesyoun** [comment]
@sreeder, I imported this api into ODM2 Web service, and tested it.
There is one error found:
In the class Results at the model.py, 
ResultTypeCV = Column('resulttypecv', ForeignKey(CVResultType.Name), nullable=False, index=True)
 Like the above, please add the column name.
When running the code with this api, I saw some "print" statements for debugging in the log window.
Please remove them if unnecessary.
